docs/practices/communication.md

```diff
@@ -0,0 +1,45 @@
+# Communication and support
+
+We organize our discussions in order to help both contributors and
+maintainers find and choose the right communication channels and have a positive experience :-)
+
+In this respect, we use:
+
+1. GitHub issues for specific discussions related to changing a repository's content
+(e.g. feature requests, bug reports).
+2. The [Discourse forum](http://discourse.jupyter.org/) for general discussions, support questions, or just as a place where we can inspire each other.
+
+## Gitter channels
+
+We use several gitter channels to have synchronous conversation across projects.
+If a conversation will likely span multiple hours, or is relevant to many people, consider opening a thread in Discourse or the `team-compass` repository instead.
+Here are the relevant Gitters:
+
+ * [The JupyterHub Gitter](https://gitter.im/jupyterhub/jupyterhub) is for
+   discussing JupyterHub, Z2JH, TLJH, etc.
+ * [The Binder Gitter](https://gitter.im/jupyterhub/binder) is for
+   discussion with Binder users about how to use Binder.
+ * [The mybinder.org-deploy gitter](https://gitter.im/jupyterhub/mybinder.org-deploy)
+   is for discussion around **operating myinder.org**.
+
+## Support bot
+
+To support our efforts in organizing the communication channels and make the
+transition easier for both developers and users, we have plugged into the repositories
+of the JupyterHub organization a [support bot](https://github.com/jupyterhub/.github/blob/master/.github/support.yml).
+
+---
+Currently, the bot will only check the following repositories:
+* [binderhub](https://github.com/jupyterhub/binderhub)
+* [jupyterhub](https://github.com/jupyterhub/jupyterhub)
+* [the-littlest-jupyterhub](https://github.com/jupyterhub/the-littlest-jupyterhub)
+---
```

Comment:
These kinds of lists are unsustainable to maintain overall I think, where its better to link to the configuration where one can see what repositories its active for.

No change requested though as its not unproblematic in this case to not be mistaken about the current state sometime in the future when this is outdated.

Reply:
I agree w you. If there's a place where this is configured I'm happy to link it in this PR. If it's more complex then let's open an issue for a future fix?

Reply:
I was thinking it was you who had written these lines now, but it was just relocated - let's not add an action point for such a trivial thing!